code/index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="This is a boilerplate for building websites"/>
+    <title>Trad Is Rad</title>
+    <link href='http://fonts.googleapis.com/css?family=Amatic+SC' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/reset.css" type="text.css" />
+    <link rel="stylesheet" href="css/style.css" type="text/css"/>
+    <!-- adding google fonts <script src="url"></script> -->
+  </head>
+
+  <body>
+    <header>
+      <div class="container">
+        <div class="logo">
+          <p>BEACHED WHALE</p>
+          <p>ADVENTURE APPAREL &amp KNICKKNACKS</p>
+        </div>
+        <nav>
+          <ul>
+            <li><a href="#">The Story</a></li>
+            <li><a href="#">Products</a></li>
+            <li><a href="#">Follow Us</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#">Cart</a></li>
+          </ul>
+        </nav>
+      </div> <!-- container -->
+    </header>
+    <section id="main">
+      <div class="container">
+
+        <div id="landing">
+          <h1 class="logo">BEACHED WHALE</h1>
+          <!--<img class="whaleLogo" src="images/whaleSample.png" alt="Company Logo">-->
+          <h2 class="subLogo">ADVENTURE APPAREL &amp KNICKKNACKS</h2>
+        </div> <!-- #landing-->
+
+        <div id="story">
+          <article class="story_text">
+						<h2><a href="#">As the Story Goes.</a></h2>
```

Comment:
Watch the indentation here. It's probably a mix of spaces and tabs - either is fine but pick one and stick with it.